ResearchKit/Common/ORKSurveyAnswerCellForScale.m

```diff
@@ -119,9 +119,7 @@ - (IBAction)sliderValueChanged:(id)sender {
 }
 
 + (CGFloat)suggestedCellHeightForView:(UIView *)view {
-    return 140.0;
+    return 244.0;
 }
 
```

Comment:
I think better to add a second `vertical` parameter to this method, so it can return different height for different layout.